src/components/CommentList/CommentList.scss

```
@@ -1 +1,14 @@
-//
+.commentList {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+  background-color: #cce2ff;
+  padding: 1em;
+  border-radius: 10px;
```

sort this
why gap 0.7rem??
```suggestion
  display: flex;
  flex-direction: column;
  gap: 0.7em;
  
  background-color: #cce2ff;
  padding: 1em;
  border-radius: 10px;
```